docs/_news/news07.md

```diff
@@ -20,14 +20,20 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Node.js, Liberty, and Spring projects. You can [enable or disable](https://github.com/eclipse/codewind/issues/1290) this feature with just a click.
+- Effortlessly [visualize data](https://github.com/eclipse/codewind/issues/977) like CPU usage, memory footprint, and web traffic on the included application monitoring dashboard at the click of a button. Codewind hosts the dashboard, so displaying the graphs won't eat into your project's resources. 
+
 **OpenShift Do (odo)**
 - You can now easily import existing projects into Codewind and [use odo](https://github.com/eclipse/codewind/issues/1115) to build and deploy projects to your OpenShift or OKD cluster.
-- Introducing [support for Java template](https://github.com/eclipse/codewind/issues/450) and Java projects can build and deploy [once configured](//permalink to landing page).
-- You no longer have to create your [docker registry secret](https://github.com/eclipse/codewind/issues/665) in Codewind.
-
+- Codewind now [supports Java templates](https://github.com/eclipse/codewind/issues/450), and Java projects can build and deploy after they have been [configured](mdt-che-odo-support.html).
 
+**Docker**
+- You can now specify any [Docker registry secret](https://github.com/eclipse/codewind/issues/1178) required by Codewind within the Codewind extension for the editor.
 
 #### List of Fixes
-- Changes to the Eclipse plugin for Spring Server [post-code generation stability for OpenAPI](https://github.com/eclipse/codewind/issues/1116)
-- Appsody binary updated from 0.5.0 to 0.5.2 in the codewind-appsody-extension
+- Changes to the Eclipse plug-in for Spring server [post-code generation stability for OpenAPI](https://github.com/eclipse/codewind/issues/1116)
```

Comment:
What about..."Improvements to OpenApi Eclipse, including fixes to ensure that the project compiles successfully after code generation."